frontend/package.json

```diff
@@ -75,7 +75,7 @@
     "msw": "^2.4.1",
     "msw-storybook-addon": "^2.0.3",
     "nyc": "^17.0.0",
-    "prettier": "^3.3.3",
+    "prettier": "^3.5.3",
```

Comment:
Why are we bumping prettier in this PR?

If we are doing that, it should be in a separate PR that explains the justification for doing so.

Please remove that from this PR and restore package-lock.json to what's on the main branch; and if we need to bump the version of prettier, let's do that in a separate PR.